saltapi/repository/proposal_repository.py

```diff
@@ -613,16 +613,14 @@ def _investigators(self, proposal_code: str) -> List[Dict[str, Any]]:
             del investigator["relevance_of_proposal"]
             del investigator["year_of_completion"]
 
-            if investigator["approved"] == 1:
+            if investigator["approval_code"] is None:
```

Comment:
Given that `investigator["approval_code"]` is a string, couldn't you just use

```python
if not investigator["approval_code"]:
    investigator["has_approved"] = None
```

Also (and most definitely not for this PR), I think the cleaner option would be to replace `has_approved` with `approval_status`, which would be either of `Approved`, `Rejected` or `No decision made yet`.